extra-admin/throttled/spec

```diff
@@ -0,0 +1,5 @@
+VER=0.9.2
+# REL=0
+
+SRCS="tbl::https://github.com/erpalma/throttled/archive/refs/tags/v$VER.tar.gz"
+CHKSUMS="sha256::01beb3e17303b11d86aeadc74d85d0ee2d50375998137dcdb5e464de4fdc4e07"
```

Comment:
Here we need to add a variable called `CHKUPDATE`, which can help us to check if the new version of application is available. You could see [here](https://github.com/AOSC-Dev/aosc-findupdate/blob/master/docs/config.zh-CN.md) for more information.